src/ORM/ResultSet.php

```diff
@@ -185,8 +184,7 @@ public function __construct(Query $query, StatementInterface $statement)
      *
      * @return object|array
      */
-    #[ReturnTypeWillChange]
-    public function current()
+    public function current(): object|array
```

Comment:
Can

Reply:
Can non-entity result sets be used in plugins? I don't think so, right?

Reply:
I haven't seen resultsets having anything besides entity or array.